cypress/e2e/gui/detalheCliente.cy.js

```diff
@@ -0,0 +1,22 @@
+/// <reference types="cypress" />
+describe('Detalhes do cliente', () => {
+    beforeEach(() => {
+        cy.visit('/')
+    })
+    it('Campo entrada de texto deve estar desabilitado', () => {
+        cy.get('tbody tr')
+            .first()  // seleciona o primeiro elemento da lista
+            .click()  // clica no primeiro elemento da lista
+        cy.get('[data-testid="name"]')
+            .should('be.disabled')
+    })
```

Comment:
```suggestion
    })

```

Deixar uma linha em branco entre cada bloco `it` também ajuda na legibilidade.

Reply:
Obrigada pela dica!